### PR TITLE
Replace std::cout/cerr with Logger calls

### PR DESCRIPTION
**Pull request by schroedtert**
Replace std::cout/std::cerr calls with logger in our code:
- replace by Logger call when it makes sense
- remove output otherwise

Note: Some third party libraries may still put there output to std::cout/cerr
Note: Some calls just replaced by stringstreams as the return value of some function could not be used by our Logger without more work, e.g. dsv output in jpsreport.

### CODE REVIEW

File: jpsreport/methods/Method_J.cpp [hunk 1]
```diff
@@ -194,8 +194,12 @@ bool Method_J::Process(
                     }
                 } else {
                     for(int i = 0; i < (int) IdInFrame.size(); i++) {
-                        std::cout << XInFrame[i] * CMtoM << "   " << YInFrame[i] * CMtoM << "   "
-                                  << IdInFrame[i] << "\n";
+                        LOG_WARNING(
```
Comment (JetteSchumann) [inline jpsreport/methods/Method_J.cpp:197]: I suggest to add text here: "Polygons could not be obtained for..."

File: libcore/src/geometry/Building.cpp [hunk 2]
```diff
@@ -859,14 +859,14 @@ const std::vector<Pedestrian *> & Building::GetAllPedestrians() const
...
-        Pedestrian * ped1 = _allPedestrians[p];
-        if(ped->GetID() == ped1->GetID()) {
-            std::cout << "Pedestrian " << ped->GetID() << " already in the room." << std::endl;
-            return;
-        }
+    if(std::find_if(
+           std::begin(_allPedestrians), std::end(_allPedestrians), [ped](const Pedestrian * other) {
+               return ped->GetID() == other->GetID();
+           }) != std::end(_allPedestrians)) {
+        LOG_ERROR("Pedestrian {} already in the romm.", ped->GetID());
```
Comment (JetteSchumann) [inline libcore/src/geometry/Building.cpp:866]: might be a warning only?

File: libcore/src/math/VelocityModel.cpp [hunk 3]
```diff
@@ -99,7 +98,7 @@ bool VelocityModel::Init(Building * building)
         if(ped->GetExitLine())
             target = ped->GetExitLine()->ShortestPoint(ped->GetPos());
         else {
-            std::cout << "Ped " << ped->GetID() << " has no exit line in INIT\n";
+            LOG_ERROR("Ped {} has no exit line in INIT", ped->GetID());
```
Comment (JetteSchumann) [inline libcore/src/math/VelocityModel.cpp:101]: warning only?

File: libcore/src/geometry/Building.cpp (hunk 2 above)
Comment (JetteSchumann) [reply libcore/src/geometry/Building.cpp:866]: "room"

File: jpsreport/methods/Method_J.cpp (hunk 1 above)
Comment (schroedtert) [reply jpsreport/methods/Method_J.cpp:197]: changed

File: libcore/src/geometry/Building.cpp (hunk 2 above)
Comment (schroedtert) [reply libcore/src/geometry/Building.cpp]: changed

File: libcore/src/math/VelocityModel.cpp (hunk 3 above)
Comment (schroedtert) [reply libcore/src/math/VelocityModel.cpp]: changed

**Pull request merged**